.github/workflows/ci.yml

```diff
@@ -11,14 +11,18 @@ on:
 jobs:
   build:
```

Comment:
shouldn't this be named `tests`? so the workflow jobs are named like `CI/Node:12-tests`